libs/zzz/ui/src/Character/card/CharacterCardEquipmentRow.tsx

```
@@ -0,0 +1,163 @@
+        width: '147px',
+        height: '129px',
+        top: '75px',
+        right: '75px',
+        background: `url(${commonDefImages('discDrive')})`,
+        backgroundSize: '100% 100%',
+        zIndex: '10',
+        transform: 'scale(1.96)',
+      }}
+    >
+      {mappedDiscs.map((discInfo: DiscInfo) => (
+        <Box>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)